src/types.ts

```diff
@@ -1,4 +1,4 @@
 export interface SelectOption {
 	label: string
-	value: string
+	value: string | string[]
```

Comment:
Cool! I think this makes sense to me. 

My only slight hesitation is that this might cause a bit of hassle for other consuming projects that were relying on this to *only* be a string. They may have to add a check now, and that could be a mild problem. But, just adding a `.toString()` might be enough, and we can see how it plays out - I'm good with merging this. Normally, I'd mark this as a breaking / major change, but we want to avoid `1.0.0` for now. Let's go ahead, and then we'll just be careful when we update the version in, say, `reggie`, or any app where it hasn't been tested yet.

Reply:
Yea it is a bit of a pickle huh? I need to reiterate that I'm not 100% convinced this is necessary and theres a certain elegance to having this only accept strings. The issue is, it does feel strange to have values represented that are effectively arrays, but we need to serialize into strings in order to make them work. The gamma example made it apparent to me, but I think its going to be apparent in other places. 

the other option here, and its maybe worse, is to make this accept an array of strings and only an array of strings. Thoughts on that @jackkoppa ?

Reply:
Ugh, have thought about this more, and yeah, there doesn't seem to be a great answer

1. In an ideal world, we wouldn't care what a consumer passed in as their options. As long as they passed in a an object with a label, they could make the `value` whatever they wanted - we just care about what to display, and then they can do what they want with the values based on events or `v-model`. It'd be great for them if they could make `value` a string, an array of strings, an object, even a function - whatever
2. However, that's not the case for this accessible select. We need to be able to set an `activeIndex`, and the only way to do that is to reliably know which value is which. We need them to be unique, and we need to be able to compare them (it also helps for us to be able to follow Vue best practices by setting a `:key`)
3. So as much as we would want *anything goes* to be OK for the `value`, it can't be


I have one idea, which may just be overkill at this point: 
* If you pass in options of the form `{ label: 'some string', value: 'another' }`, you're good - you may proceed on your merry way & things will just work, because we'll match those strings as if they're unique & comparable to one another
* if you want to pass in anything else, then you need to *either*:
   * add a `uniqueId` to each option - we'll use that, then
   * pass in a function as a prop that can take in each option & always *return* a uniqueId - perhaps it's `option => option.value.join('_')`

And we could then set the value of `SelectOption { value: any }`, and just add JS checks to make sure of what its type was. THIS IS PROBABLY WAY OVERKILL.

But, we're trying to be smart in advance of this having wider usage 🤷

Reply:
Work to be done in a future ticket!